### PR TITLE
status: add new http status API for get meta regions.

### PR DESCRIPTION
**Pull request by hicqu**
Add a new API( **/regions/meta** ) for get meta regions.
The return JSON is:
```json
[
    {
        "leader": {
            "id": 3,
            "store_id": 1
        },
        "peers": [
            {
                "id": 3,
                "store_id": 1
            }
        ],
        "region_epoch": {
            "conf_ver": 1,
            "version": 1
        },
        "region_id": 2
    }
]
```
**Meta** means those TiKV keys which start with `m`.

@siddontang , PTAL, thanks.

### CODE REVIEW

**Comment by coocood** [comment]
@hicqu 
Please add test.

File: server/region_handler_test.go [hunk 1]
```diff
@@ -169,6 +169,15 @@ func (ts *TidbRegionHandlerTestSuite) TestGetRegionByIDWithError(c *C) {
 	defer resp.Body.Close()
 }
 
+func (ts *TidbRegionHandlerTestSuite) TestRegionsFromMeta(c *C) {
+	ts.startServer(c)
+	defer ts.stopServer(c)
+	resp, err := http.Get("http://127.0.0.1:10090/regions/meta")
+	c.Assert(err, IsNil)
+	defer resp.Body.Close()
+	c.Assert(resp.StatusCode, Equals, http.StatusOK)
```
Comment (coocood) [inline server/region_handler_test.go]: Why not verify the body?

**Comment by shenli** [comment]
@hicqu Please address the comment.

File: server/region_handler_test.go (hunk 1 above)
Comment (hicqu) [reply server/region_handler_test.go]: Now we do verify whether the body is a valid json array or not.

Comment (coocood) [reply server/region_handler_test.go]: Why not verify if it is region meta?

Comment (hicqu) [reply server/region_handler_test.go]: Fixed. PTAL, thanks!

**Comment by coocood** [comment]
LGTM

**Comment by hicqu** [comment]
@disksing @shenli , PTAL, thanks.

**Comment by disksing** [comment]
LGTM.

**Pull request merged**